### PR TITLE
concurrent `importshare` test

### PR DESCRIPTION
**Pull request by lwin-kyaw**
add two new unittests for `importShare` request
- subsequent requests
- concurrent requests
- rebased with `master` branch

### CODE REVIEW

File: src/helpers/nodeUtils.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ export async function retrieveOrImportShare(params: {
         verifierId: verifierParams.verifier_id,
         network,
         clientId,
-        enable_gating: "true",
+        enableGating: "true",
```
Comment (metalurgical) [inline src/helpers/nodeUtils.ts:154]:
```suggestion
        enablegating: "true",
```

File: src/torus.ts [hunk 2]
```diff
@@ -63,23 +70,16 @@ class Torus {
...
     if (!network) throw Error("Please provide a valid network in constructor");
     this.ec = new EC("secp256k1");
     this.serverTimeOffset = serverTimeOffset || 0; // ms
     this.network = network;
     this.clientId = clientId;
-    this.allowHost = allowHost;
+    this.allowHost = allowHost || `${SIGNER_MAP[network]}/api/allow`;
     this.enableOneKey = enableOneKey;
-    this.legacyMetadataHost = legacyMetadataHost;
+    this.legacyMetadataHost = legacyMetadataHost || METADATA_MAP[network as TORUS_LEGACY_NETWORK_TYPE];
```
Comment (metalurgical) [inline src/torus.ts:82]: Bug here, what if in parameters, network is sapphire and this is nil?

File: test/aqua.test.ts [hunk 3]
```diff
@@ -57,7 +55,7 @@ describe("torus utils aqua", function () {
     const verifierDetails = { verifier, verifierId: TORUS_TEST_EMAIL };
     const { torusNodeEndpoints, torusNodePub } = await TORUS_NODE_MANAGER.getNodeDetails(verifierDetails);
     const result1 = (await torus.getUserTypeAndAddress(torusNodeEndpoints, torusNodePub, verifierDetails)) as TorusPublicKey;
-    expect(result1.finalKeyData.evmAddress).to.equal("0xDfA967285AC699A70DA340F60d00DB19A272639d");
+    expect(result1.finalKeyData.evmAddress).to.equal("0x79F06350eF34Aeed4BE68e26954D405D573f1438");
```
Comment (metalurgical) [inline test/aqua.test.ts:58]: Revert this, previous value is correct.

File: test/sapphire_devnet.test.ts [hunk 4]
```diff
@@ -421,6 +417,96 @@ describe("torus utils sapphire", function () {
...
+        torusIndexes: nodeDetails.torusIndexes,
+        torusNodePub: nodeDetails.torusNodePub,
+        verifier: TORUS_TEST_VERIFIER,
+        verifierId: { verifier_id: email },
+        token,
+        privHex,
+      };
+    });
+
+    const outputs = await Promise.all(
```
Comment (metalurgical) [inline test/sapphire_devnet.test.ts:477]: I think this still ultimately resolves sequentially, not so?

**Comment by himanshuchawla009** [comment]
deprecated, closing this

**Pull request closed without merging**